README.md

```diff
@@ -25,4 +25,31 @@ go build -o sst cmd/main.go
 ./sst [sample-dir]
 ```
 
-Make sure to authorize the gcloud SDK and set a default project before running this program.
+Make sure to authorize the gcloud SDK and set a default project and Cloud Run region before running this program. A
+default Cloud Run region can be set by setting the `run/region` gcloud property.
+
+### README parsing
+If you'd like, make sure to include the following comment code tag immediately preceding code blocks to customize how
+the program should build and deploy your sample:
+
+```text
+[//]: # ({sst-run-linuxmacos})
```

Comment:
This, in my understanding is one syntax for a line comment in Markdown. It's the same syntax used here: https://raw.githubusercontent.com/googleapis/java-bigtable/master/README.md , for a similar purpose. 

However, formal documentation for Markdown comments has been hard to find. But it appears there are multiple syntaxes for line comments.

Reply:
This syntax is widely supported and probably the right choice here, but comment syntax in Markdown isn't standardized, unfortunately. FYI, this SO thread has a lot of info: https://stackoverflow.com/questions/4823468/comments-in-markdown

Reply:
SG, but probably worth mentioning that in some fashion in the README

Reply:
To clarify, we should clarify what the user needs to do - is it a commented line? which kind of comment do we support? Can it be added somewhere else? We want to minimize the risk of a false positive with this.

Reply:
Okay, should we require code tags follow this regular expression pattern?:

```
\[//\]:\s+#\s+\(\s*{sst-run-linuxmacos}\s*\)
```

This would require a comment in the form of the `[//] # (comment)` syntax. We could also require that a code tag line following this format _immediately_ precede a code block.

Reply:
To clarify, I think given the ambiguity of the comment syntax I agree with your initial decision to use a looser requirement. However, you should make sure that what you are looking for is clearly defined in the README. So something like:

> When parsing the README for custom build/deploy instructions, SST will include any commands inside a code fence that is immediately preceded by a line containing `{sst-run-unix}`. You can use a markdown comment syntax (e.g. `[//] # ({sst-run-unix})`) to include this line without making it visible when rendered.

This makes it clear that:
1. The line only has to contain `{sst-run-unix}`
2. There can't be any empty lines between the line containing the tag and the code block.
3. They can use `[//] #`, but the exact format is up to them. 

You may also which to clarify the following points:
* Are multiline commands supported?
* Is bash syntax supported (`cd` or `export VAR=VAL`)?

Reply:
README updated, thank you.